src/planner.ts

```diff
@@ -115,7 +115,7 @@ export function geometryToGeoJSON(
           coordinates: [entrance.lon, entrance.lat],
         },
         properties: {
-          color: "#00ffff00",
+          color: "#00ffff",
```

Comment:
Perhaps even better to remove the line as opacity is 0 anyway.

Reply:
It was not acceptable input.